plugins/generator-1.19.4/forge-1.19.4/templates/plant/placed_plantgen.json.ftl

```diff
@@ -7,7 +7,7 @@
             "count": ${data.frequencyOnChunks}
         },
         </#if>
-        <#if (data.plantType == "normal" && data.staticPlantGenerationType == "Flower") || (data.plantType == "double" && data.doublePlantGenerationType == "Flower") || data.plantType == "growapable">
+        <#if (data.plantType == "normal" || data.plantType == "double") && data.generationType == "Flower" || data.plantType == "growapable">

```

Comment:
I would prefer if && would be in another pair of parentheses as it might be more clear for programmers not knowing ||&& priority rules, applies to all cases here